src/app/shared/nav/nav.component.html

```diff
@@ -23,7 +28,7 @@
           <ul class="navbar-nav">
             <li class="nav-item">
               <div class="nav-link">
-                Welcome {{currentUser.getValue()?.firstName + ' ' + currentUser.getValue()?.lastName}}
+                <a class="nav-link" [routerLink]="['/users', currentUser.getValue()?.id]">Welcome {{currentUser.getValue()?.firstName + ' ' + currentUser.getValue()?.lastName}}</a>
```

Comment:
you're async'ing the currentUser$ observable above, you can declare it as a variable inside the template by adding as <variable_name> something like this:

if (currentUser$ | async as currentUser) then you can use this currentUser variable in the template and directly access it's fields, like currentUser.id etc.

Reply:
I could only get this to work in for loops
![image](https://github.com/user-attachments/assets/3dc4e54b-8346-4e32-93c1-523c75dcb3df)